.github/icon_checklist.md

```diff
@@ -0,0 +1,24 @@
+Hey! Thanks for the contribution. While there is no reviewer, you can do a self-review. This will help to fix the reasons why the icons may not be suitable for Lawnicons. 
+
+### Canvas and sizes [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#canvas--sizes)
+1. Canvas: `192×192px`.
+2. Non-square icons: the long side of the icons should be `160px`.
```

Comment:
```suggestion
2. Non-square icons: the length of the icons should be `160px`.
```

Reply:
> the length of the icons should be `160px`.

The requirement is that the width or height should be 160px. In my understanding, "length" refers only to width.